src/gui/addnewtorrentdialog.cpp

```diff
@@ -300,6 +300,8 @@ bool AddNewTorrentDialog::loadTorrentImpl()
     m_ui->labelHashData->setText(m_hash);
     setupTreeview();
     TMMChanged(m_ui->comboTTM->currentIndex());
+    if (!m_torrentInfo.hasRootFolder())
+        m_ui->keepTopLevelFolderCheckBox->setEnabled(false);
```

Comment:
I think you can use `m_ui->keepTopLevelFolderCheckBox->setEnabled(m_torrentInfo.hasRootFolder());`
It looks better to me.

Reply:
done